protobufs/py/setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = p4runtime
+description = Python bindings for P4Runtime protocol
+long_description = file: README.md
+long_description_content_type = text/markdown; charset=UTF-8
+url = https://github.com/p4lang/p4runtime
+author = P4 API Working Group
+author_email = p4-api@lists.p4.org
+license = Apache-2.0
+license_file = LICENSE
+classifiers =
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+
+[options]
+packages = find:
+platforms = any
+python_requires = >=3.6
+setup_requires =
+    setuptools_scm
+install_requires =
+    protobuf >= 3.6.1
```

Comment:
Once the stratum-deps [PR 41](https://github.com/ipdk-io/stratum-deps/pull/41) is merged, this should be upgraded to use 

```
    protobuf >= 4.25.0
    grpcio >= 1.59.2
    googleapis-common-protos >= 1.61.0
```

Also, can safely update line 19 to use minimum Python 3.8 (the protobuf/grpc requirements state that it needs minimum Python 3.8).

Tested the changes with these values

Reply:
Have you established that the protobuf 4.25.0 is the minimum version needed to communicate with this version of stratum? Or that grpcio and googleapis-common-protos must be updated to the latest version? (We are currently using an older version of googleapis, BTW, and we only use use a tiny portion of it.)

If older versions continue to work, then we should stick with the current minimums. Forcing the user to make an update that affects their entire platform, without a clear need to do so, is kind of intrusive. We're not the only Python users.

Reply:
Im not sure if I can test every combination of different set of libraries.

* Python 3.8 is the current lowest version of Python that has not EOL'd. It is also the minimum required version by gRPC.
* gRPC -- I think I tried something lower than 1.59.2, but not every version is available under pip, but I 1.59.2, the latest stable version is available.
* protobuf -- 1.59.2 works with 4.25.0, which has been tested. Looking at the history, protobuf went from 3.20.x to 4.21.x in May 2022 and it was a breaking change which upped the major version. We could try 4.21.x as the minimum version to check if that works
* googleapi-common-tools - I picked the latest number. Havent researched any history of this project

Reply:
I'm not contesting Python 3.8. I updated this yesterday. I'm talking about the three modules mentioned in the first paragraph.

The only googleapi protobufs we use are `rpc/code.proto` and `rpc/status.proto`. The versions we're using are dated 2020. The versions in googleapis-common-protos 1.61.0 are dated 2022. They differ only in comments. This means that 1.61.0 is a perfectly good minimum version. There is no need to change it.

Reply:
It's not just the versions of the .proto files. It's also the code that the associated version of the protobuf compiler generates. I wonder how tightly coupled they are to the gRPC and Protobuf versions. Pfui.